Improved Documentation for Image-Filter-Application (#55)
Blaze-Stars/Image-Filter-Application: README.md

# Image-Filter-Application
### `An Application to apply different Filters to different BMPs Images`
## Table of Contents

1) [An overview](#project)
2) [Building Image-Filter-Application](#building)
3) [Running Image-Filter-Application](#running)

## Project

> This Project is Associated with Department of Integrated MCA Utkal Uniersity, Bhubaneshwar, Odisha.

Reviewer: ```suggestion
This project is associated with the Department of Integrated MCA, Utkal University, Bhubaneshwar, Odisha.
```

## Building

> 

Reviewer 2: Create a separate section or division from line 14 to 16 and label it as dependencies and prerequisite

> To `build Image-Filter-App` Make sure To install make in your System and g++ compiler.<br>

Reviewer: ```suggestion
To `build Image-Filter-App` make sure to install `make` in your system and `g++ compiler`.<br>
```

Make installation in windows -> [link](https://www.technewstoday.com/install-and-use-make-in-windows/)<br>

> Make installation in Ubuntu -> [link](https://askubuntu.com/questions/161104/how-do-i-install-make) 

Reviewer: ```suggestion
- Make installation in windows -> [link](https://www.technewstoday.com/install-and-use-make-in-windows/)<br>

- Make installation in Ubuntu -> [link](https://askubuntu.com/questions/161104/how-do-i-install-make) 

```

> To Build `Image-Filter-App` go to your project directory and Open our Terminal and just type.

Reviewer: ```suggestion
To build `Image-Filter-App`, go to your project directory and open your `Terminal` and type.
```

```shell
make
```
Boom! You have successfully build `Image-Filter-Application`
## Running

> + Currently `Image-Filter-Application` supports 5 Filters, This Filters can be used using following Flags below.<br>

Reviewer: ```suggestion
+ Currently `Image-Filter-Application` supports `5 Filters`, these filters can be used using the following flags below.<br>
```

> + We have several example Images to test these Filters stored in [imgDB](https://github.com/Blaze-Stars/Image-Filter-Application/tree/main/img-db)

Reviewer: ```suggestion
+ We have several examples images to test these filters stored in [imgDB](https://github.com/Blaze-Stars/Image-Filter-Application/tree/main/img-db)
```

| Filters | Flag |
| ------- | ----- |
| GrayScale | -g |
| Blur | -b |
| Reflect | -r |
| Edge | -e |
| Sepia | -s |

> To Use any of the Filters :

Reviewer: ```suggestion
To use any of the filters :
```

```shell
./filter -g img-db/<pickAnyImage> <YourOutputFileName>.bmp
```
### NOTE
```

> The Above example shows output for grayscale filter, you can change the flags and image to apply any other Filter 

Reviewer: ```suggestion
The above example shows output for the grayscale filter, you can change the flags and image to apply any other filter 
```
